chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+%------------------------------------------------
+
+\begin{fullwidth}
+
+% Chapter overview
+The tasks described in this chapter have many different names.
+Some say data cleaning, others data munging, or maybe data wrangling.
+All mean transforming raw data into a more convenient format for your intended use.
+Although original data typically requires more extensive data cleaning than secondary data,
+it is broadly agreed that this is, on average, the most time-consuming step in a project's data work \citep{dasu2003exploratory}.
+This chapter describes the different tasks involved in the cleaning process,
+and outlines the workflow our team developed to make the process efficient and transparent.
+
+% What is data cleaning
+This chapter describes all tasks needed to make newly acquired data ready for analysis.
+The first section will guide on making your data \textit{tidy},
+so the relationship between rows and columns in a data set is well-defined.
```

Comment:
```suggestion
so the relationship between rows and columns in a dataset is well-defined.
```
data set => dataset

Reply:
argh, I did this a lot! thanks for catching it!